cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -997,7 +1055,7 @@ private Expr getStmtResultExpr(Stmt stmt) {
 }
 
 /**
- * A C/C++ this expression.
+ * The C++ `this` pointer.
```

Comment:
Why is it better to describe `this` as a pointer rather than an expression?

(I agree with replacing "C/C++" with just "C++" here).

Reply:
Calling it a pointer is just simpler (and the way most C++ programmers think about `this`).